### PR TITLE
project finished

### PR DESCRIPTION
**Pull request by bouaik**
## Private events
- user can create events
- user can invite other users to his events

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  has_many :event_attendances, foreign_key: :attende_id
+  has_many :attended_events, through: :event_attendances, source: :attendances
+
+  def to_s
+    name
+  end
+
+  # scope :upcoming_events, -> { events.where('date >= ?', Date.today) }
+  # scope :previous_events, -> { events.where('date < ?', Date.today) }
+
```
Comment (kobiyoyo) [inline app/models/user.rb:18]: Let's remove this commented code

File: app/views/events/_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<%= form_with(model: event, local: true) do |form| %>
+  <% if event.errors.any? %>
```
Comment (uryelah) [inline app/views/events/_form.html.erb:2]: - [ ] Please refactor view logic, conditionals and loops, to helper methods.

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -0,0 +1,34 @@
+<nav class="navbar navbar-expand-lg navbar-light bg-secondary">
+  <%= link_to "Private Events", root_path, class: "navbar-brand"%>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+
+  <div class="collapse navbar-collapse" id="navbarSupportedContent">
+    <ul class="navbar-nav mr-auto">
+      <% if current_user %>
```
Comment (uryelah) [inline app/views/layouts/_header.html.erb:9]: - [ ] Please refactor view logic, conditionals and loops, to helper methods.

File: app/views/users/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,24 @@
+<h1 class="text-center my-4">Sign up</h1>
+<%= form_with(model: @user, local: true) do |f| %>
+
+   <% if @user.errors.any? %>
```
Comment (uryelah) [inline app/views/users/new.html.erb:4]: - [ ] Please refactor view logic, conditionals and loops, to helper methods.

**Pull request merged**